build/images/scripts/restart_sandbox

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+CRICTL_VERSION="v1.26.0"
+wget https://github.com/kubernetes-sigs/cri-tools/releases/download/$CRICTL_VERSION/crictl-$CRICTL_VERSION-linux-amd64.tar.gz
+tar zxvf crictl-$CRICTL_VERSION-linux-amd64.tar.gz -C /usr/local/bin
+rm -f crictl-$CRICTL_VERSION-linux-amd64.tar.gz
+
+# Remove rules of other CNI in CHAIN CNI-HOSTPORT-DNAT
+# CHAIN CNI-HOSTPORT-DNAT is created by portmap for ingress controllers. It will not be deleted
+# even if the CNI is removed, so we delete it manually.
+iptables -t nat -F CNI-HOSTPORT-DNAT || true
+chains=$(iptables -t nat -L | grep CNI-DN | grep -v "antrea" | awk '{print $2}')
+for chain in $chains; do
+    iptables -t nat -X "$chain" || true
+done
+
+pods=$(crictl pods | grep -v "POD ID" | awk '{print $1}')
+for pod in $pods; do
+    network_type=$(crictl inspectp $pod | jq -r .status.linux.namespaces.options.network)
+    if [ "$network_type" == "POD" ]; then
+        crictl stopp $pod && crictl rmp $pod
```

Comment:
does it need to remove the pod? I feel we should still let container runtime manage its lifecycle.

Reply:
Do you mean `crictl rmp $pod` is necessary to trigger another CNI event?

Reply:
Yes. In `crictl` command, the `crictl ps` command will not show the sandbox ID of the Pods. Although we use `crictl inspect $CONTAINER` to get the sandbox ID, but when we run the command `crictl rm $SANDBOX_ID`, it will return an error as follows:
```bash
E0109 03:15:53.227320    6356 remote_runtime.go:415] "ContainerStatus from runtime service failed" err="rpc error: code = NotFound desc = an error occurred when try to find container \"8a9ed4da2c445f4ff90158cea8b6c71efe9ad74e53153189a2d5d4aa9bd72b89\": not found" containerID="8a9ed4da2c445f4ff90158cea8b6c71efe9ad74e53153189a2d5d4aa9bd72b89"
ERRO[0000] rpc error: code = NotFound desc = an error occurred when try to find container "8a9ed4da2c445f4ff90158cea8b6c71efe9ad74e53153189a2d5d4aa9bd72b89": not found
FATA[0000] unable to remove container(s)
```

If we remove the workload container directly, it will not trigger another CNI event but just recreate the container in-place. That's why I use `crictl rmp` command.